lib/features/label-editing/cmd/UpdateLabelHandler.js

```diff
@@ -77,7 +53,7 @@ export default function UpdateLabelHandler(modeling, textRenderer, bpmnFactory)
 
     setLabel(label, text, labelTarget !== label);
 
-    ensureInternalLabelDi(element, text);
+    // ensureInternalLabelDi(element, text);
```

Comment:
We'd want to get rid of this line, right?

Reply:
Yeah, my bad